src/clue/app-config.json

```diff
@@ -245,9 +246,6 @@
       "content": "curriculum/comparing-and-scaling/comparing-and-scaling.json",
       "guide": "curriculum/comparing-and-scaling/cas-teacher-guide.json"
     },
-    "dataflow-example": {
```

Comment:
We should keep the dataflow example in `app-config.json`. If you merge master into this branch before merging this branch back into master I think this problem should be solved. Let me know if you want me to do this.

Reply:
yeah, I'm sure I started this branch a while ago. If you wouldn't mind sorting out the sequencing so all things don't get clobbered I would appreciate it.

Reply:
Alright I think you're ready to merge with master now.